composer.json

```diff
@@ -36,11 +36,11 @@
         "ramsey/collection": "^1.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "^7",
+        "phpunit/phpunit": "^8",
```

Comment:
The `phpunit 8.x` version requires `php-7.2` version at least.
And it's fine to do `PHPUnit` version upgrading.